OpenRA.Mods.Common/Widgets/ColorMixerWidget.cs

```diff
@@ -162,7 +162,8 @@ public override bool HandleMouseInput(MouseInput mi)
 		/// </summary>
 		public void Set(Color color)
 		{
-			var (_, h, s, _) = color.ToAhsv();
```

Comment:
I don't think we want this one.

Reply:
After more investigation, yeah, this is incorrect